aztfq/aztfq.go

```diff
@@ -8,17 +8,18 @@ import (
 	"strings"
 
 	"github.com/magodo/azure-rest-api-bridge/ctrl"
+	"github.com/magodo/azure-rest-api-bridge/mockserver/swagger"
 )
 
-func BuildLookupTable(input []byte) (LookupTable, error) {
+func BuildLookupTable(input []byte, removeArraySymbol bool) (LookupTable, error) {
```

Comment:
Instead of introducing new properties for new options, can we do something like below:

```
type Option struct {
	// ImplicitArrayIndex makes the array index to be implicit, e.g. "foos.*.id" -> "foos.id".
	ImplicitArrayIndex bool
}

func BuildLookupTable(input []byte, opt *Option) (LookupTable, error) {
```